src/components/WebsiteField.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import Checkbox from "@material-ui/core/Checkbox";
+import FormGroup from "@material-ui/core/FormGroup";
+import FormControl from "@material-ui/core/FormControl";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Switch from "@material-ui/core/Switch";
+import TextField from "@material-ui/core/TextField";
+import { DeleteButton } from "../styling/Icons";
+
+export const WebsiteField = (props) => {
+  const [label, setLabel] = useState("Disable interruptions");
+
+  const switchLabel = () => {
+    setLabel(
+      label === "Disable interruptions"
+        ? "Enable interruptions"
+        : "Disable interruptions"
+    );
+  };
+
+  const handleCheckboxChange = () => {
+    props.changeCheckState(props.website);
+  };
+
+  const handleInputChange = (event) => {
+    disableOtherCheckboxes();
+    props.updateInterval(event.target.value);
+  };
+
+  const disableOtherCheckboxes = () => {
+    props.disableCheckboxes(props.website);
+  };
```

Comment:
If the function is just one line, there's no benefit to having this extra "wrapper" function instead of just calling `props.disableCheckboxes(props.website);` directly :)